code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -171,23 +190,35 @@
 		if(ALIEN_BODYPART,LARVA_BODYPART) //aliens take double burn //nothing can burn with so much snowflake code around
 			burn *= 2
 
-	var/can_inflict = max_damage - get_damage()
-	if(can_inflict <= 0)
-		return FALSE
+	var/wounding_type = (brute > burn ? WOUND_BRUTE : WOUND_BURN)
+	var/wounding_dmg = max(brute, burn)
+	if(wounding_type == WOUND_BRUTE && sharpness)
+		wounding_type = WOUND_SHARP
+	// i know this is effectively the same check as above but i don't know if those can null the damage by rounding and want to be safe
+	if(owner && wounding_dmg > 4 && wound_bonus != CANT_WOUND)
+		// if you want to make tox wounds or some other type, this will need to be expanded and made more modular
+		// handle all our wounding stuff
+		check_wounding(wounding_type, wounding_dmg, wound_bonus, bare_wound_bonus)
 
+	var/can_inflict = max_damage - get_damage()
 	var/total_damage = brute + burn
-
-	if(total_damage > can_inflict)
+	if(total_damage > can_inflict && total_damage > 0) // TODO: the second part of this check should be removed once disabling is all done
 		brute = round(brute * (can_inflict / total_damage),DAMAGE_PRECISION)
 		burn = round(burn * (can_inflict / total_damage),DAMAGE_PRECISION)
 
+	if(can_inflict <= 0)
```

Comment:
so do stamina hits to limbs that have full brute damage do nothing?

Reply:
what about stamina hits to limbs that are 1 point of brute damage away from maxing out their brute damage?

Reply:
yes